repository-wildfly/pom.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm.contrib</groupId>
+    <artifactId>workitems</artifactId>
+    <version>7.8.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>repository-wildfly</artifactId>
+  <packaging>war</packaging>
+  <name>Repository WildFly</name>
+  <description>Workitems WildFly Repository</description>
+  <organization>
+    <name>jBPM dev team</name>
+  </organization>
+
+  <properties>
+    <version.org.wildfly>13.0.0.Final</version.org.wildfly>
```

Comment:
let's use 11 as this is the version we use for all our web apps

Reply:
we should use the version property from kie-parent top level parent IMHO. Now it seems this is forgotten when we did upgrades tpo 14.0.0.Final. Is it intentionally left to use 11.0.0.Final now?